src/main/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/api/GetConfigurationApiAction.java

```diff
@@ -54,7 +54,7 @@ public GetConfigurationApiAction(final Settings settings, final Path configPath,
 			final AdminDNs adminDNs, final IndexBaseConfigurationRepository cl, final ClusterService cs,
             final PrincipalExtractor principalExtractor, final PrivilegesEvaluator evaluator, ThreadPool threadPool, AuditLog auditLog) {
 		super(settings, configPath, controller, client, adminDNs, cl, cs, principalExtractor, evaluator, threadPool, auditLog);
-		controller.registerHandler(Method.GET, "/_opendistro/_security/api/configuration/{configname}", this);
```

Comment:
why are we removing configuration endpoint?  Can you please provide more details for these change? 

Can you please provide result for 
/_opendistro/_security/api/configuration/{configname}  
(like /_opendistro/_security/api/configuration/roles,  /_opendistro/_security/api/configuration/config 

also please provide UTs result.

Reply:
@deprecated 
Will close the PR.